packages/editor/src/components/properties/PostProcessingSettingsEditor.tsx

```diff
@@ -90,7 +97,7 @@ const EffectsOptions: Partial<EffectOptionsType> = {
     intensity: { propertyType: PropertyTypes.Number, name: 'Intensity', min: 0, max: 10, step: 0.01 },
     fade: { propertyType: PropertyTypes.Number, name: 'Fade', min: -1, max: 1, step: 0.01 },
     resolutionScale: { propertyType: PropertyTypes.Number, name: 'Resolution Scale', min: -10, max: 10, step: 0.01 },
-    kernelSize: { propertyType: PropertyTypes.Number, name: 'Kerne Size', min: 1, max: 5, step: 1 },
+    kernelSize: { propertyType: PropertyTypes.KernelSize, name: 'KernelSize' },
```

Comment:
The min max and step for this was slightly correct. I actually think there is a KernelSelect we could use for this with values from 0-5. There are others that can use it too.